.github/workflows/upgrade-python-requirements.yml

```diff
@@ -0,0 +1,30 @@
+name: Upgrade Python Requirements
+
+on:
+  schedule:
+    - cron: "0 0 * * 1"
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "Target branch against which to create requirements PR"
+        required: true
+        # If copying this template manually, you must provide your default branch name
+        # in quotes, such as 'master'
+        default: "main"
+
+jobs:
+  call-upgrade-python-requirements-workflow:
+    uses: openedx/.github/.github/workflows/upgrade-python-requirements.yml@master
+    with:
+    # If copying manually, also provide your default branch name in quotes here
+      branch: ${{ github.event.inputs.branch || "main" }}
+      # optional parameters below; fill in if you'd like github or email notifications
+      # user_reviewers: ""
+      # team_reviewers: ""
```

Comment:
Not a blocker, but you may want to add a reviewing team unless y'all are just watching the repo for all changes and will see the PRs.